kagami/webui/src/api/api.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
```

Comment:
Please update `package.json` when you add or upgrade a dependency. I strongly recommend turning on the code checking feature of your IDE to avoid similar problems.
![image](https://github.com/user-attachments/assets/66d2e8b0-1665-41c5-95ea-a8d2ce094e74)